Assets/Examples/AdditiveScenes/Prefabs/Player.prefab

```diff
@@ -233,7 +233,7 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   sceneId: 0
-  serverOnly: 0
+  serverOnly: 1
```

Comment:
Does the Player prefab need to be server only?

Reply:
Good spot, no! I'll change that.